packages/server/lib/modes/interactive-ct.js

```diff
@@ -1,6 +1,71 @@
-const serverCt = require('@packages/server-ct')
+const chalk = require('chalk')
+const human = require('human-interval')
+const _ = require('lodash')
+const Debug = require('debug')
+
 const { getBrowsers } = require('../browsers/utils')
 const errors = require('../errors')
+const browsers = require('../browsers')
+const { openProject } = require('../open_project')
+const Updater = require('../updater')
+
+const debug = Debug('cypress:server:interactive-ct')
+
+const registerCheckForUpdates = () => {
+  const checkForUpdates = (initialLaunch) => {
+    Updater.check({
+      initialLaunch,
+      testingType: 'component',
+      onNewVersion: _.noop,
+      onNoNewVersion: _.noop,
+    })
+  }
+
+  setInterval(() => checkForUpdates(false), human('60 minutes'))
+  checkForUpdates(true)
+}
+
+// Partial because there are probably other options that are not included in this type.
+// projectRoot: string
+// args: LaunchArgs
+const start = async (projectRoot, args) => {
+  if (process.env['CYPRESS_INTERNAL_ENV'] === 'production') {
+    registerCheckForUpdates()
+  }
+
+  debug('start server-ct on ', projectRoot)
+
+  // add chrome as a default browser if none has been specified
+  return browsers.ensureAndGetByNameOrPath(args.browser)
+  .then((browser) => {
+    const spec = {
+      name: 'All Specs',
+      absolute: '__all',
+      relative: '__all',
+      specType: 'component',
+    }
+
+    const options = {
+      browsers: [browser],
+    }
+
+    debug('create project')
+
+    return openProject.create(projectRoot, args, options)
+    .then((project) => {
+      debug('launch project')
+
+      return openProject.launch(browser, spec, {
+        onBrowserClose: () => {
+          /* eslint-disable no-console */
+          console.log(chalk.blue('BROWSER EXITED SAFELY'))
+          console.log(chalk.blue('COMPONENT TESTING STOPPED'))
```

Comment:
Should these logs be displayed to the user or should they be `debug` logs?

Reply:
This PR was purely a "move the things around" PR, so I guess whatever is there was the original intention. 

That said, I'd say `debug` is probably a better fit for this. I will make this change.

Reply:
I did it: https://github.com/cypress-io/cypress/pull/18054/commits/599e113c0bd93f4c592e73a3ffd709040f0be14d